massa-models/src/config/constants.rs

```diff
@@ -192,7 +192,7 @@ pub const MAX_RNG_SEED_LENGTH: u32 = PERIODS_PER_CYCLE.saturating_mul(THREAD_COU
 //
 
 /// Max message size for bootstrap
-pub const MAX_BOOTSTRAP_MESSAGE_SIZE: u32 = 1048576000;
+pub const MAX_BOOTSTRAP_MESSAGE_SIZE: u32 = u32::MAX; //1048576000;
```

Comment:
Will you be able to reduce this limit after more thourough testing on sizes you're doing currently?

Reply:
I made a test with the max sizes everywhere in the message generation.
Hope I didn't miss anything that could go beyond the values I used (even if I uses constants defined in the file)
The new version got an updated value for the max message size